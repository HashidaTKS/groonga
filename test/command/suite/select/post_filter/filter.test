table_create Items TABLE_NO_KEY
column_create Items price COLUMN_SCALAR UInt32

load --table Items
[
{"price": 100},
{"price": 150},
{"price": 200},
{"price": 250},
{"price": 300}
]

select Items \
  --filter "price >= 150" \
  --post_filter "price <= 250"
