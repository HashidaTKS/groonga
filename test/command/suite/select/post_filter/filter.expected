table_create Items TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Items price COLUMN_SCALAR UInt32
[[0,0.0,0.0],true]
load --table Items
[
{"price": 100},
{"price": 150},
{"price": 200},
{"price": 250},
{"price": 300}
]
[[0,0.0,0.0],5]
select Items   --filter "price >= 150"   --post_filter "price <= 250"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        3
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "price",
          "UInt32"
        ]
      ],
      [
        2,
        150
      ],
      [
        3,
        200
      ],
      [
        4,
        250
      ]
    ]
  ]
]
